Maple2.Server.Game/Commands/PlayerCommand.cs

```diff
@@ -457,6 +458,35 @@ private void Handle(InvocationContext ctx, string tab) {
                 }
             }
         }
+
+        private class PrintInventoryCommand : Command {
```

Comment:
Feels like this is more of a debug command, no?

Reply:
uhh sure